<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualArcWeight</class>
 <widget class="QWidget" name="ManualArcWeight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leMimg">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Load activations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbFind">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../qresources.qrc">
         <normaloff>:/Images/icons/folder_open.svg</normaloff>:/Images/icons/folder_open.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="sbAdj">
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Adjacency relation radius</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../qresources.qrc"/>
 </resources>
 <connections/>
</ui>
